### PR TITLE
Flexible rendering options for the access control widget

### PR DESCRIPTION
**Pull request by zachmullen**

### CODE REVIEW

**Comment by zachmullen** [comment]
re: COVALIC. @cpatrick @mgrauer

File: clients/web/src/stylesheets/widgets/accessWidget.styl [hunk 1]
```diff
@@ -18,16 +20,15 @@
       color #d0b822
 
 .g-ac-list-title
-  background-color #666
+  background-color #888
```
Comment (manthey) [inline clients/web/src/stylesheets/widgets/accessWidget.styl]: Just an idle question: I've noticed a trend to making web pages lower contrast.  It doesn't matter much here, but is there some logic to it?

Comment (jeffbaumes) [inline clients/web/src/stylesheets/widgets/accessWidget.styl]: Not sure, but I get the feeling that `'#000'` is like the ALL-CAPS of the 90's - just too loud. Soft and friendly is where it's at.

Comment (jbeezley) [inline clients/web/src/stylesheets/widgets/accessWidget.styl]: I'm patiently waiting for the geocities look to come back in style.

**Comment by manthey** [comment]
LGTM

**Pull request merged**